website/source/intro/getting-started/build-image.html.md

```diff
@@ -200,4 +198,269 @@ image was pretty useless in this case (nothing was changed about it), this page
 should've given you a general idea of how Packer works, what templates are and
 how to validate and build templates into machine images.
 
+## Some more examples:
+
+### Another Linux Example, with provisioners:
+Create a file named `welcome.txt` and add the following:
+```
+WELCOME TO PACKER!
+```
+
+Create a file named `example.sh` and add the following:
+```
+#!/bin/bash
+echo "hello
+```
+
+Set your access key and id as environment variables, so we don't need to pass them in through the command line:
+```
+export AWS_ACCESS_KEY_ID=MYACCESSKEYID
+export AWS_SECRET_ACCESS_KEY=MYSECRETACCESSKEY
+```
+
+Now save the following text in a file named `firstrun.json`:
+
+```
+{
+    "variables": {
+        "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+        "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+        "region":         "us-east-1"
+    },
+    "builders": [
+        {
+            "access_key": "{{user `aws_access_key`}}",
+            "ami_name": "packer-linux-aws-demo-{{timestamp}}",
+            "instance_type": "t2.micro",
+            "region": "us-east-1",
+            "secret_key": "{{user `aws_secret_key`}}",
+            "source_ami": "ami-fce3c696",
```

Comment:
I see you're trying to demonstrate what a json with a `source_ami` looks like, but I think it'll be confusing when it's out of date, and at the very least be a maintenance headache. We could call out that you could use a source_ami instead of a filter, but I think all our docs should use filters.

Reply:
Fair.  I'll call that out below.